markdownlint/TOP005_blanksAroundMultilineHtmlTags/tests/TOP005_test.md

```diff
@@ -86,6 +86,14 @@ This line above the closing tag is not a blank line nor a code block delimiter,
 <% } %>
 ```
 
+```ruby
+if ruby?
+  html_fragment = <<~HTML
+    <p>Does not flag when used in ruby code blocks</p>
+  HTML
+end
+```
```

Comment:
As per above, just adding the JS block example.

````suggestion
```ruby
if ruby?
  html_fragment = <<~HTML
    <p>Does not flag when used in ruby code blocks</p>
  HTML
end
```

```javascript
const htmlString = `
  <p>Does not flag when used in JavaScript code blocks, e.g. template literals.</p>
`;
```
````